javascript/builtins/Intl.json

```diff
@@ -132,6 +133,293 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "calendar": {
+            "__compat": {
+              "description": "`\"calendar\"`",
```

Comment:
@queengooborg Should we add a [`key_parameter_` prefix](https://github.com/search?q=repo%3Amdn%2Fbrowser-compat-data%20%2F%22key_parameter_%2F&type=code) or [`_value` suffix](https://github.com/search?q=repo%3Amdn%2Fbrowser-compat-data+%2F_value%22%3A+%5C%7B%2F+-path%3Aschemas&type=code) here?

Given that this method just takes a single parameter, I felt like it's unnecessary.

Reply:
I'd say yes, using `key_parameter_calendar` will make it more self-explanatory and machine-parsable!